nixos/modules/services/hardware/udisks2.nix

```
@@ -19,7 +18,18 @@ in
 
     services.udisks2 = {
 
-      enable = mkEnableOption (lib.mdDoc "udisks2, a DBus service that allows applications to query and manipulate storage devices");
+      enable = mkEnableOption (mdDoc "udisks2, a DBus service that allows applications to query and manipulate storage devices");
+
+      mountRoot = mkOption {
```

So I suggest:


```suggestion
      mountOnMedia = mkOption {
```

This is literally the title of the link stated as relevant documentation.